File: app/src/main/java/com/bogdanorzea/popularmovies/model/response/CreditsResponse.java
package com.bogdanorzea.popularmovies.model.response;

import com.bogdanorzea.popularmovies.model.object.Cast;
import com.squareup.moshi.Json;

import java.util.List;

public class CreditsResponse {

    @Json(name = "id")
    public int id;
    @Json(name = "cast")
    public List<Cast> cast = null;

}

File: app/src/main/java/com/bogdanorzea/popularmovies/model/response/CreditsResponse.kt
package com.bogdanorzea.popularmovies.model.response

import com.bogdanorzea.popularmovies.model.`object`.Cast
import com.squareup.moshi.Json

data class CreditsResponse(@Json(name = "id") var id: Int = 0,
                           @Json(name = "cast") var cast: List<Cast>? = null)
